src/main/resources/findologic_20.xsd

```diff
@@ -39,10 +45,11 @@
             <xs:element type="allAttributesType" name="allAttributes" minOccurs="1" maxOccurs="1"/>
             <xs:element type="allKeywordsType" name="allKeywords" minOccurs="1" maxOccurs="1"/>
             <xs:element type="salesFrequenciesType" name="salesFrequencies" minOccurs="1" maxOccurs="1"/>
-            <xs:element type="usergroupsType" name="usergroups" minOccurs="1" maxOccurs="1"/>
+            <xs:element type="groupsType" name="groups" minOccurs="1" maxOccurs="1"/>
```

Comment:
Do you have an argument telling why we have to change the name here!?

Reply:
We had a discussion with the stakeholder about this and decided to rename usergroups to groups.
The different naming just leads to confusion on customer side, was the main argument.

Reply:
I consider this as resolved. Please reopen if there are more questions regarding this